test_community/acquisition/test_latent_information_gain.py

```diff
@@ -32,16 +33,22 @@ def test_initialization(self):
         self.assertEqual(self.acquisition_function.num_samples, 10)
         self.assertEqual(self.acquisition_function.model, self.model)
 
-    def test_acquisition_shape(self):
-        self.model(self.model.train_X, self.model.train_Y)
-        lig_score = self.acquisition_function.forward(candidate_x=self.candidate_x)
-        self.assertTrue(torch.is_tensor(lig_score))
-        self.assertEqual(lig_score.shape, (1, 5))
-
-    def test_acquisition_kl(self):
-        self.model(self.model.train_X, self.model.train_Y)
-        lig_score = self.acquisition_function.forward(candidate_x=self.candidate_x)
-        self.assertGreaterEqual(lig_score.mean().item(), 0)
+    def test_acqf(self):

```

Comment:
you can use the `mock_optimize_context_manager` to speed up these tests if they are slow: https://github.com/pytorch/botorch/blob/ed2e5c04b88ad6678fff1c01fed756bda4345130/botorch/test_utils/mock.py#L30